docs/api-reference.md

```diff
@@ -226,6 +289,48 @@ class App {
 
 ```
 
+
+`getAuth(): FirebaseAuthState` : Returns the client's current Authentication State.
+
+Sample Usage:
+
+```ts
+constructor(public auth: FirebaseAuth) {
+        this.authState = auth.getAuth();
+        auth.subscribe((state: FirebaseAuthState) => {
+            this.authState = state;          
+        });        
+    }
+
+    get authenticated(): boolean {        
+        return this.authState !== null;
+    }
+```
+
+`logout(): void`: Deletes the authentication token issued by Firebase and signs user out. See [Auth.signOut()](https://firebase.google.com/docs/reference/js/firebase.auth.Auth#signOut) in the Firebase API reference.
```

Comment:
@katowulf Thanks for update. I was unaware that this is an open bug against firebase. I pasted the contents as per your suggestion. Hope that's ok.